frontend/src/booking/calendar.vue

```diff
@@ -0,0 +1,254 @@
+<template>
+    <div @mouseup="resetDrag"
+         @click="resetDrag"
+         style="position: relative; height: 100%; width: 100%">
+      <keep-alive>
+        <full-calendar ref="qcalendar"
```

Comment:
Mmm yes I see, if this has ref `qcalendar`, then the other one has ref `acalendar`? If so, can you update the names on both to be more representative of what each ref actually is?

Reply:
sure.. will update

Reply:
went with bookingcal and agendacal